python/src/main/parser.py

```diff
@@ -10,3 +10,7 @@ def create_person(row):
 def create_item(row):
     person_email, name, price, priority, hyperlink = row[0], row[1], row[2], row[3], row[4]
     return Item(person_email, name, price, priority, hyperlink)
+
+def parse_list_of_items(lst):
```

Comment:
try not to include the type of data structure in naming. Ie

Bad :  def parse_list_of_items(lst):
Good : def parse_items(items):

Reply:
duly noted